src/borg/constants.py

```diff
@@ -60,6 +60,10 @@
 # chunker params for the items metadata stream, finer granularity
 ITEMS_CHUNKER_PARAMS = (15, 19, 17, HASH_WINDOW_SIZE)
 
+# operating mode of the files cache (for fast skipping of unchanged files)
+DEFAULT_CACHE_MODE_UI = 'ctime,size,inode'
+DEFAULT_CACHE_MODE = 'cis'  # == CacheMode(DEFAULT_CACHE_MODE_UI)
```

Comment:
didn't want to import CacheMode from helpers into constants to avoid cyclic imports.

that these 2 actually correspond is asserted elsewhere.